blog-posts/deploying-nextjs-13-with-amplify-cdk/deploying-nextjs-13-with-amplify-cdk.md

```diff
@@ -0,0 +1,281 @@
+---
+published: false
+title: 'Deploying Next.js 13 with Amplify CDK'
+cover_image: './assets/cover-image.png'
+description: 'How to deploy a Next.js app on AWS Amplify with a CDK construct'
+tags: aws, cdk, nextjs, amplify
+series:
+canonical_url:
+---
+
+> 📜 We detail step by step how to deploy a Next.js app with a Amplify CDK construct, avoiding you all the pains.
```

Comment:
Tantôt "We", tantôt "I" -> choisir.
C'est vrai que "we" sonne un peu bizarre, en tout cas en première ligne de l'article.

Reply:
Merci pour ce retour ! J'ai changé un peu l'histoire pour plus de tournures passives et en disant qu'on était plusieurs sur le sujet dans l'intro.